### PR TITLE
Add numpad emote hotkey bindings

### PR DESCRIPTION
**Pull request by Stolij**
## Status

**READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description
This pull request's main goal is to add numpad keybindings to EmoteHotkeys.as. It allows players who have a numpad to utilize it to access more emote hotkeys.

## Steps to Test or Reproduce
To use the extended EmoteHotkeys, just press any key on your numpad (between 1-9)

### Files modified
EmoteHotkeys.as
EmoteBinderMenu.as

EmoteBindings.cfg

### Used branch
The branch I created is called 'more-emote-hotkeys'

### CODE REVIEW

File: Rules/CommonScripts/EmoteBinderMenu.as [hunk 1]
```diff
@@ -98,22 +98,31 @@ void ShowEmotesMenu(CPlayer@ player)
...
-			read_emote(cfg, "emote_1", Emotes::attn),
-			read_emote(cfg, "emote_2", Emotes::smile),
-			read_emote(cfg, "emote_3", Emotes::frown),
-			read_emote(cfg, "emote_4", Emotes::mad),
-			read_emote(cfg, "emote_5", Emotes::laugh),
-			read_emote(cfg, "emote_6", Emotes::wat),
-			read_emote(cfg, "emote_7", Emotes::troll),
-			read_emote(cfg, "emote_8", Emotes::disappoint),
-			read_emote(cfg, "emote_9", Emotes::ladder)
+                    read_emote(cfg, "emote_1", Emotes::attn),
```
Comment (1bardesign) [inline Rules/CommonScripts/EmoteBinderMenu.as]: how come this is indented with spaces when the rest of the file is not?

File: Entities/Common/Emotes/EmoteBindings.cfg [hunk 2]
```diff
@@ -13,3 +13,13 @@ emote_6 = wat;
...
+
+emote_10 = flex;
+emote_11 = down;
+emote_12 = smug;
+emote_13 = left;
+emote_14 = okhand;
+emote_15 = right;
+emote_16 = thumbsup;
+emote_17 = up;
+emote_18 = thumbsdown;
```
Comment (1bardesign) [inline Entities/Common/Emotes/EmoteBindings.cfg]: would love a newline at the end of the file 😉

**Comment by Stolij** [comment]
I have just fixed the indent of EmoteBinderMenu.as, and added a newline at the end of EmoteBindings.cfg

File: Entities/Common/Emotes/EmoteHotkeys.as [hunk 3]
```diff
@@ -34,6 +44,16 @@ void onInit(CBlob@ this)
 	emote_8 = read_emote(cfg, "emote_8", Emotes::disappoint);
 	emote_9 = read_emote(cfg, "emote_9", Emotes::ladder);
 
+    emote_10 = read_emote(cfg, "emote_10", Emotes::flex);
```
Comment (furai) [inline Entities/Common/Emotes/EmoteHotkeys.as]: This line still looks off with indentation. Editors let you reindent whole files. You don't have to do it manually.

Comment (Stolij) [reply Entities/Common/Emotes/EmoteHotkeys.as]: Oops, I totally forgot that! I'll give it a try.

**Comment by Stolij** [comment]
Okay, I have ran the auto-indentation tool in VSCode, indentation *should* be all tabs now.

**Pull request merged**